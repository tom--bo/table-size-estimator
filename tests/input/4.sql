create table `inlist` (
  `id` int(11) default null,
  `accountoid` int(11) default null,
  `dt` datetime default null,
  `x` decimal(18,4) default null
) engine=innodb default charset = utf8mb4 ;
