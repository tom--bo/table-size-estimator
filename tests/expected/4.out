Input Table Definition: (Please type ^d to end input) 
successfully ended
------
1 row max size = 32 bytes .
1 row Average size = 32 bytes .
